scripts/Editor/DllManipulatorEditor.cs

```diff
@@ -21,6 +21,10 @@ public class DllManipulatorEditor : Editor
             "If true, native functions will be mocked only in assembly that contains DllManipulator (usually Assembly-CSharp)");
         private static readonly GUIContent ONLY_IN_EDITOR = new GUIContent("Only in editor",
             "Whether to run only inside editor (which is recommended).");
+        private static readonly GUIContent ENABLE_IN_EDIT_MODE = new GUIContent("Enable in Edit Mode",
+            "Should the DLLs also be mocked in edit mode. " +
+            "Turning this off when not needed improves performance when entering edit mode. " +
+            "Changed are currently only visible on the next time edit mode is entered.");
```

Comment:
Users might not know (and so I don't know) what exactly means "the next time edit mode is entered". Does it mean I have to click 'play' and then 'stop'? Also should be 'Changes' not 'Changed'

Reply:
Well the change is only effective the next time `OnEnable` is called, which happens when we enter playmode or edit mode (ie not playmode). So yes you need to `play` then `stop`.